tools/mpremote/mpremote/main.py

```diff
@@ -355,17 +357,23 @@ def load_user_config():
     # Create empty config object.
     config = __build_class__(lambda: None, "Config")()
     config.commands = {}
-
-    # Get config file name.
-    path = os.getenv("XDG_CONFIG_HOME")
-    if path is None:
-        path = os.getenv("HOME")
-        if path is None:
-            return config
-        path = os.path.join(path, ".config")
-    path = os.path.join(path, _PROG)
+    # use $XDG_CONFIG_HOME,$HOME $env:USERPROFILE% or $env:APPDATA
+    path = None
+    for env_var in ("XDG_CONFIG_HOME", "HOME", "USERPROFILE", "APPDATA"):
```

Comment:
Sorry didn't think this through earlier, but I think there's actually zero chance APPDATA is ever going to be used, since a windows system without USERPROFILE would be pretty much broken. Likewise any msys/cygwin like system will have HOME. So seems better to not use APPDATA at all.

Reply:
The advantage of using APPDATA is that it roams across all my computers for free  ( no need for a .files repo ) 
so for my N=3 , there is a 100% change that it is used. 
USERPROFILE is checked first for those who prefer a old-school manual approach and will prefer a `%USERPROFILE%\.mpremote` folder over `%APPDATA%\.mpremote`

Reply:
So just to be clear: you do not have the USERPROFILE environment variable?